src/components/Dashboard.scss

```diff
@@ -93,3 +95,79 @@
     }
   }
 }
+
+.main-layout__header {
+  .main-header__logo {
+    span {
+      font-size: 0.9rem;
+    }
+  }
+}
```

Comment:
Please move this styles definitions to assigned stylesheet file (`MainHeaderComponent`)

```SCSS
.main-header__logo span {
  font-size: 0.9rem;
}
```

This should do the job.

Reply:
Correct. Thanks